src/proposals/20241114/SparkBase_20241114.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { Base, SparkPayloadBase } from 'src/SparkPayloadBase.sol';
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { CCTPForwarder } from 'xchain-helpers/forwarders/CCTPForwarder.sol';
+
+import {
+    ControllerInstance,
+    ForeignControllerInit,
+    RateLimitData,
+    MintRecipient
+} from 'lib/spark-alm-controller/deploy/ControllerInit.sol';
+
+/**
+ * @title  Nov 14, 2024 Spark Base Proposal
+ * @notice Activate Spark Liquidity Layer
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/14-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25466
+ * Vote:   https://vote.makerdao.com/polling/Qmf955yA
+ */
+contract SparkBase_20241114 is SparkPayloadBase {
+
+    address internal constant FREEZER = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;  // Gov. facilitator multisig
+    address internal constant RELAYER = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+
+    function execute() external {
+        // --- Activate Mainnet Controller ---
```

Comment:
Is it called "Mainnet" on Base? or should be "Foreign"?

Reply:
yes should be foreign, corrected